Dockerfile Arm64 npm ci command error

Error msg:
```
#0 7.799 npm ERR! gyp ERR! find Python 
#0 7.799 npm ERR! gyp ERR! find Python Python is not set from command line or npm configuration
#0 7.799 npm ERR! gyp ERR! find Python Python is not set from environment variable PYTHON
#0 7.799 npm ERR! gyp ERR! find Python checking if "python3" can be used
#0 7.799 npm ERR! gyp ERR! find Python - "python3" is not in PATH or produced an error
#0 7.799 npm ERR! gyp ERR! find Python checking if "python" can be used
#0 7.799 npm ERR! gyp ERR! find Python - "python" is not in PATH or produced an error
#0 7.799 npm ERR! gyp ERR! find Python 
#0 7.799 npm ERR! gyp ERR! find Python **********************************************************
#0 7.799 npm ERR! gyp ERR! find Python You need to install the latest version of Python.
#0 7.799 npm ERR! gyp ERR! find Python Node-gyp should be able to find and use Python. If not,
#0 7.799 npm ERR! gyp ERR! find Python you can try one of the following options:
#0 7.800 npm ERR! gyp ERR! find Python - Use the switch --python="/path/to/pythonexecutable"
#0 7.800 npm ERR! gyp ERR! find Python   (accepted by both node-gyp and npm)
#0 7.800 npm ERR! gyp ERR! find Python - Set the environment variable PYTHON
#0 7.800 npm ERR! gyp ERR! find Python - Set the npm configuration variable python:
#0 7.800 npm ERR! gyp ERR! find Python   npm config set python "/path/to/pythonexecutable"
#0 7.800 npm ERR! gyp ERR! find Python For more information consult the documentation at:
#0 7.800 npm ERR! gyp ERR! find Python https://github.com/nodejs/node-gyp#installation
#0 7.800 npm ERR! gyp ERR! find Python **********************************************************
#0 7.800 npm ERR! gyp ERR! find Python 
#0 7.800 npm ERR! gyp ERR! configure error 
#0 7.800 npm ERR! gyp ERR! stack Error: Could not find any Python installation to use
#0 7.800 npm ERR! gyp ERR! stack     at PythonFinder.fail (/usr/local/lib/node_modules/npm/node_modules/node-gyp/lib/find-python.js:330:47)
#0 7.800 npm ERR! gyp ERR! stack     at PythonFinder.runChecks (/usr/local/lib/node_modules/npm/node_modules/node-gyp/lib/find-python.js:159:21)
#0 7.800 npm ERR! gyp ERR! stack     at PythonFinder.<anonymous> (/usr/local/lib/node_modules/npm/node_modules/node-gyp/lib/find-python.js:202:16)
#0 7.800 npm ERR! gyp ERR! stack     at PythonFinder.execFileCallback (/usr/local/lib/node_modules/npm/node_modules/node-gyp/lib/find-python.js:294:16)
#0 7.800 npm ERR! gyp ERR! stack     at exithandler (node:child_process:427:5)
#0 7.800 npm ERR! gyp ERR! stack     at ChildProcess.errorhandler (node:child_process:439:5)
#0 7.800 npm ERR! gyp ERR! stack     at ChildProcess.emit (node:events:513:28)
#0 7.800 npm ERR! gyp ERR! stack     at ChildProcess._handle.onexit (node:internal/child_process:289:12)
#0 7.800 npm ERR! gyp ERR! stack     at onErrorNT (node:internal/child_process:476:16)
#0 7.800 npm ERR! gyp ERR! stack     at process.processTicksAndRejections (node:internal/process/task_queues:82:21)
#0 7.800 npm ERR! gyp ERR! System Linux 5.15.49-linuxkit
#0 7.800 npm ERR! gyp ERR! command "/usr/local/bin/node" "/usr/local/lib/node_modules/npm/node_modules/node-gyp/bin/node-gyp.js" "rebuild"
#0 7.800 npm ERR! gyp ERR! cwd /usr/src/app/node_modules/bufferutil
#0 7.800 npm ERR! gyp ERR! node -v v18.16.0
#0 7.800 npm ERR! gyp ERR! node-gyp -v v9.3.1
#0 7.800 npm ERR! gyp ERR! not ok
```


Fix with line nums:
```
18: RUN apk add --update python3 make g++ && rm -rf /var/cache/apk/*
19: RUN npm ci
```

--- a/Dockerfile
+++ b/Dockerfile
@@ -15,7 +15,8 @@ WORKDIR /usr/src/app
 
 # install dependencies to the image
 COPY src/frontend/vite.config.ts src/frontend/tsconfig.node.json src/frontend/tsconfig.json src/frontend/package.json src/frontend/package-lock.json /usr/src/app/
-RUN npm ci --legacy-peer-deps
+RUN apk add --update python3 make g++ && rm -rf /var/cache/apk/*
+RUN npm ci
 
 ENV PATH /usr/src/app/node_modules/.bin:$PATH
 

